tests/plugins/data/test-plugin/test_plugin-1.0.dist-info/entry_points.txt

```diff
@@ -1,2 +1,4 @@
-[conda]
-conda-test-plugin=test_plugin.plugin
+[test_plugin]
```

Comment:
I believe this change fixes the noisy logging that would bleed over into other tests:

```
WARNING conda.plugins.manager:load_entrypoints(86): Could not load conda plugin `conda-test-plugin`:

No module named 'package_that_does_not_exist'
```

Reply:
cc @kalawac